workshops/models.py

```diff
@@ -108,7 +108,7 @@ class Person(AbstractBaseUser, PermissionsMixin):
     email       = models.CharField(max_length=STR_LONG, unique=True, null=True, blank=True)
     gender      = models.CharField(max_length=1, choices=GENDER_CHOICES, null=True, blank=True)
     may_contact = models.BooleanField(default=True)
-    airport     = models.ForeignKey(Airport, null=True, blank=True)
+    airport     = models.ForeignKey(Airport, null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
I figured that we didn't need to forbid deletion of Airport if some Person links to it, because we don't manipulate airports at all.

I can change `on_delete` to `on_delete=models.PROTECT`, but this would raise a question: what would we do with Event.site and Event.organizer fields, that have `on_delete=models.SET_NULL`, too?